docs/standard/parallel-programming/dataflow-task-parallel-library.md

```diff
@@ -118,7 +118,7 @@ ms.author: "ronpet"
  Execution blocks call a user-provided delegate for each piece of received data. The TPL Dataflow Library provides three execution block types: <xref:System.Threading.Tasks.Dataflow.ActionBlock%601>, <xref:System.Threading.Tasks.Dataflow.TransformBlock%602?displayProperty=nameWithType>, and <xref:System.Threading.Tasks.Dataflow.TransformManyBlock%602?displayProperty=nameWithType>.  
   
 #### ActionBlock(T)  
- The <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> class is a target block that calls a delegate when it receives data. Think of a <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object as a delegate that runs asynchronously when data becomes available. The delegate that you provide to an <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object can be of type <xref:System.Action> or type `System.Func\<TInput, Task>`. When you use an <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object with <xref:System.Action>, processing of each input element is considered completed when the delegate returns. When you use an <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object with `System.Func\<TInput, Task>`, processing of each input element is considered completed only when the returned <xref:System.Threading.Tasks.Task> object is completed. By using these two mechanisms, you can use <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> for both synchronous and asynchronous processing of each input element.  
+ The <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> class is a target block that calls a delegate when it receives data. Think of a <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object as a delegate that runs asynchronously when data becomes available. The delegate that you provide to an <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object can be of type <xref:System.Action> or type `System.Func<TInput, Task>`. When you use an <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object with <xref:System.Action>, processing of each input element is considered completed when the delegate returns. When you use an <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> object with `System.Func<TInput, Task>`, processing of each input element is considered completed only when the returned <xref:System.Threading.Tasks.Task> object is completed. By using these two mechanisms, you can use <xref:System.Threading.Tasks.Dataflow.ActionBlock%601> for both synchronous and asynchronous processing of each input element.  
```

Comment:
Actually, it appears, @imba-tjd, that the `System.Action` delegate is incorrect as well; it should be \<xref:System.Action %601>. There are two occurrences of it in this paragraph.

Reply:
Thanks, done. I also made two more changes.